project/guitar-changer/out/production/guitar-changer/migrations/V00001_Create_changer.sql

```diff
@@ -0,0 +1,86 @@
+CREATE SCHEMA IF NOT EXISTS project;
+
+CREATE TABLE IF NOT EXISTS users
+(
+    id           uuid DEFAULT gen_random_uuid() PRIMARY KEY,
+    first_name   VARCHAR(32) NOT NULL,
+    last_name    VARCHAR(32) NOT NULL,
+    phone_number VARCHAR(32) NOT NULL,
+    password     VARCHAR(16) NOT NULL,
+    address      TEXT,
+    role         VARCHAR(16) NOT NULL
+);
+
+
+CREATE TABLE IF NOT EXISTS change_types
+(
+    change_type SERIAL PRIMARY KEY,
+    description text NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS products
+(
+    id           uuid    DEFAULT gen_random_uuid() PRIMARY KEY,
+    timestamp    TIMESTAMP WITHOUT TIME ZONE,
+    user_id      uuid REFERENCES users,
+    price        DOUBLE PRECISION NOT NULL ,
+    is_closed    BOOLEAN default FALSE,
+    change_type  INT NOT NULL ,
+    change_value DOUBLE PRECISION,
+    change_wish  VARCHAR(32),
+    FOREIGN KEY (change_type) REFERENCES change_types (change_type)
+);
+
+CREATE TABLE IF NOT EXISTS guitars
+(
+    id               uuid DEFAULT gen_random_uuid() PRIMARY KEY,
+    product_id       uuid REFERENCES products on delete cascade,
+    brand            VARCHAR(32) NOT NULL,
+    model            VARCHAR(32) NOT NULL,
+    description      TEXT,
+    year             INT,
+    country          VARCHAR(32),
+    pick_ups         VARCHAR(32),
+    fingerboard_wood VARCHAR(32),
+    sound_board      VARCHAR(32),
+    media_name       VARCHAR(32)
+);
+
+CREATE TABLE IF NOT EXISTS pedals
+(
+    id          uuid DEFAULT gen_random_uuid() PRIMARY KEY,
+    product_id  uuid REFERENCES products on delete cascade,
+    brand       VARCHAR(32) NOT NULL,
```

Comment:
CREATE TABLE IF NOT EXISTS brand
(
    id   BIGSERIAL PRIMARY KEY,
    name VARCHAR(255) NOT NULL
);

Reply:
brand to products 
from products ref to brand